test-suite/rust-runtime-api/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
```

Comment:
Shouldn't we specify [`publish = false`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish--field-optional)?